styles.css

```diff
@@ -90,3 +90,406 @@ header nav a {
   list-style: none;
   color: inherit;
 }
+
+.projects {
+  margin: 0 20px;
+  position: relative;
```

Comment:
- [ ] Kindly layout the elements in this section using grid `display: grid`

Reply:
Thanks I will sure implement that right away

Reply:
👍🏽 Good now.